draft-law-moq-cat4moqt.md

```diff
@@ -106,56 +106,72 @@ The "moqt" claim is defined by the following CDDL:
 
 ~~~~~~~~~~~~~~~
 $$Claims-Set-Claims //= (moqt-label => moqt-value)
-moqt-label = XXX TODO - how do we register this?
-moqt-value = [ + moqt-object ]
+moqt-label = TBD_MOQT
+moqt-value = [ + moqt-limit ]
+moqt-limit = [ moqt-actions, moqt-ns-match, moqt-track-match ]
+moqt-actions = moqt-action | [ + moqt-action ]
+moqt-action = int
+moqt-ns-match = match
+moqt-track-match = match
+
+/ match defined in CTA-5007-B 4.6.1, repeated here for convenience /
+match = {
+  ? exact-match ^ => tstr,
+  ? prefix-match ^ => tstr,
+  ? suffix-match ^ => tstr,
+  ? contains-match ^ => tstr,
+  ? regex-match ^ => [ tstr, * tstr ],
+  ? sha256-match ^ => bstr,
+  ? sha512-256-match ^ => bstr
+}
+
+exact-match = 0
+prefix-match = 1
+suffix-match = 2
+contains-match = 3
```

Comment:
may be we should limit the match possibilities to exact and prefix match. Also a CAT question, can't prefix match also perform exact match as well ?

Reply:
Not sure I understand the question. A prefix match on "/bob" matches "/bob", "/bobby", "/bob/stuff". An exact match would only match the first.